commands/general/stats.js

```diff
@@ -14,7 +14,7 @@ module.exports.load = (client) => {
 
       if (args.length >= 1) {
         // fqName: "fully qualified name"
-        let fqName = args[0].split('#')
+        let fqName = args.join(' ').trim().split('#')
```

Comment:
Supposedly you can just not type the spaces and Discord will just deal. But maybe this is good too.